haystack/components/embedders/backends/sentence_transformers_sparse_backend.py

```diff
@@ -3,11 +3,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
```

Comment:
See my comment above. I phrased it incorrectly in the original issue.

Let's leave this file unchanged.